packages/bpk-component-image/src/BpkImage.js

```diff
@@ -56,22 +75,23 @@ Image.defaultProps = {
   hidden: false,
 };
 
-class BpkImage extends React.Component {
-  constructor(props) {
-    super(props);
+class BpkImage extends Component<BpkImageProps> {
+  onImageLoad: Function;
+  placeholder: ?HTMLElement;
+  static defaultProps: {};
 
+  constructor(props: BpkImageProps) {
+    super(props);
     this.onImageLoad = this.onImageLoad.bind(this);
-
-    this.placeholder = null;
   }
 
-  onImageLoad() {
+  onImageLoad = (): mixed => {
```

Comment:
Why mixed? This doesn't return anything so should be `void`

Reply:
Also can't Flow infer this?

Reply:
the callback is from userland, it can return something or nothing,

Reply:
Yeah @k0nserv this is to allow folk to do inline arrow functions with implicit returns ...

Reply:
But this isn't the callback it's a method on the component.

Reply:
Ah in this particular example yeah - got confused because there are two components in this file. 👍